refnx/reduce/test/test_batch_reduce.py

```diff
@@ -19,19 +19,19 @@ class TestReduce:
     @pytest.mark.usefixtures("no_data_directory")
     @pytest.fixture(autouse=True)
     def setup_method(self, tmpdir, data_directory):
-        self.pth = pjoin(data_directory, "reduce")
+        self.pth = Path(data_directory) / "reduce"
         self.cwd = os.getcwd()
 
-        self.tmpdir = tmpdir.strpath
+        self.tmpdir = Path(tmpdir.strpath)
```

Comment:
Maybe switch tmpdir to `tmp_path` while at it? https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-fixture
(also no need to go via `strpath` then